### ISSUE TITLE
The save cards function is not working.

### ISSUE THREAD

**Issue by ntijoh-linus-bjorken** [itggot-TE4/Yabs#729]
### Description
The "spara alla bilder" and "spara bild" buttons on admin/cards is not working. It should save the pictures in db but it dosen't.

### Expected behavior
When I press the "spara alla bilder button" or the "spara bild" the pictures should be saved to db.
## 🔬 To Reproduce
1. Go to admin
2. Click on cards
3. Upload a picture
4. press "spara bild" or "spara alla bilder"
5. go to db file in vscode
6. See error

**Comment by ntijoh-daniel-berg** [comment]
Duplicate of #628?

**Comment by ntijoh-linus-bjorken** [comment]
> 

No, this is for when you just save the pictures not downloading it.

**Comment by ntijoh-daniel-berg** [comment]
Seem like #628 also wants to save to backend though.

**Comment by ntijoh-linus-bjorken** [comment]
Yes, but in #628 it's when you download the picture. In this issue it's when you just wanna save to backend and not download.

**Comment by ntijoh-daniel-berg** [comment]
👍
